docs/openapi/howto/archboard_feedback.md

```diff
@@ -0,0 +1,152 @@
+│  │  ├─ package1/
+│  │  │  ├─ Microsoft.MyService/ (swaggers for package 1)
+│  │  │  ├─ readme.md
+│  │  ├─ package2/
+│  │  │  ├─ Microsoft.MyService/ (swaggers for package 2)
+│  │  │  ├─ readme.md
+```
+
+## Rename an operation
+
+Operation name are directly OperationID in Swagger. Just rename the OperationID directly.
+
+Note: If your operation contains an `_`, just change the second part as the first is an operation group.
+
+``` json
+ "operationId": "SomethingToDoOperation",
+ ...
+ "operationId": "MyGroup_SomethingElseToDoOperation",
+```
+becomes
+``` json
+ "operationId": "BuildTheThing",
+ ...
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+
+## Rename an operation group (or rename a sub-client)
+
+Operation groups are the first part of an OperationID in Swagger, before the `_` character. To rename an operation group, you need to rename all OperationID that
+starts with the operation group prefix (there is likely many of them).
+
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+becomes
+``` json
+ "operationId": "NewGroup_BuildSomethingElse",
+```
+
+## Need one client to have all operations directly on it (no sub-client)
+
+Don't use operation groups in your OperationID (remove them if they exist). For any operation with an `_` character, remove the first part.
+
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+becomes
+``` json
+ "operationId": "BuildSomethingElse",
+```
+
+## Need one client/builder with sub-clients
+
+-	Use operation groups, and name them as you want your subclients to be called. Operation group can be create in Swagger by using the `_` character in the OperationID 
+(`OperationGroupName_OperationName`)
+-	For C#, use `single-top-level-client: true`
+
+For instance, this:
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+
+Will create a subclient for MyGroup, that will have an operation called BuildSomethingElse.
+In various languages this could looks like (pseudo-code):
+```
+MyServiceClient(endpoint, credentials).MyGroup().BuildSomethingElse() // C#
+
+MyServiceClient(endpoint, credentials).my_group.build_something_else() # Python
+```
+
+## Need two or more service clients (regardless of if some needs subclients or not)
+
+-	Make sure you don't have one Swagger with operations that are designed to be in two different clients.
+  Split Swagger files in multiple swagger files if necessary: clients should correspond to a clear set of Swagger files.
+-	Define tags in `Readme.md` for each client input files. Use the `batch` autorest option to create the two clients:-
+
+```yaml
+batch:
+- tag-client1
+- tag-client2
+```
+
+-	For each individual clients, follow the previous guidance on single clients. You may have options specific to only one tag for one client.
+
+## Rename a model name
+
+Just rename the model name directly in Swagger. Since this name is not use in JSON serialization, there is no need to use specific extensions.
+
+## Rename a model attribute
+
```

Comment:
If the API is not yet GA, the first choice should be to rename the property in the API.  This will require changes to the service code, but is preferred to doing renaming only in the SDKs.

Reply:
updated